src/ui/simulator/windows/inspector/data.cpp

```diff
@@ -33,7 +33,7 @@ namespace Window
 {
 namespace Inspector
 {
-InspectorData::InspectorData(Antares::Data::Study& s) : study(s), empty(true)
+InspectorData::InspectorData(Antares::Data::Study::Ptr s) : study(s), empty(true)
```

Comment:
Why replacing & with a smart pointer ?

Reply:
To avoid de-referencing that pointer using `std::shared_ptr<T>::get`. Exposing the address of the managed object can lead to invalid addresses if that object is destroyed when its ref counter reaches 0.